tests/dummy/app/pods/view/renderers/models/checkbox-array.js

```diff
@@ -6,6 +6,19 @@ export default {
         type: 'string'
       },
       type: 'array'
+    },
+    serviceAffecting: {
+      items: {
+        type: 'string'
```

Comment:
That's kind of my point, you shouldn't override possible values in the view, unless they're factored into the validation.